OrbitCaptureClient/include/OrbitCaptureClient/CaptureEventProcessor.h

```diff
@@ -57,6 +83,14 @@ class CaptureEventProcessor {
   absl::flat_hash_set<uint64_t> tracepoint_hashes_seen_;
   void SendTracepointInfoToListenerIfNecessary(
       const orbit_grpc_protos::TracepointInfo& tracepoint_info, const uint64_t& hash);
+
+  absl::flat_hash_map<int32_t, std::map<uint64_t, orbit_grpc_protos::GpuJob>>
+      tid_to_submission_time_to_gpu_job_;
```

Comment:
Would it be more accurate to call this `tid_to_pre_submission_time_to_gpu_job_` (note the "pre") or did I get it wrong? Or do you think it's better to omit it?

Reply:
Makes sense!